extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -238,27 +236,28 @@ function createDicomWebApi(dicomWebConfig, UserAuthenticationService) {
           'Unable to query for SeriesMetadata without StudyInstanceUID'
         );
       }
+      const studyResponse = await nlApi.get("http://localhost:8000/api/studies/?uid=" + StudyInstanceUID);
```

Comment:
@adinhodovic any chance we could set this URL from the nl-pacs side instead of localhost?